src/app/(home-page)/youtube/page.jsx

```diff
@@ -33,4 +33,4 @@ const YoutubePage = () => (
 
 export default YoutubePage;
 
-export const revalidate = 60;
+export const revalidate = false;
```

Comment:
Isn't `false` the default?

https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate